control/controlclient/direct.go

```diff
@@ -317,6 +318,21 @@ func (c *Direct) SetNetInfo(ni *tailcfg.NetInfo) bool {
 	return true
 }
 
+// SetNetInfo stores a new tka head value for next update.
+// True is returned if the tka head changed.
```

Comment:
// It reports whether the TKA head changed.

(use TKA in docs as it's an acronym)